### ISSUE TITLE
Cannot call find_by_external_id

### ISSUE THREAD

**Issue by stephenbinns** [chartmogul/chartmogul-ruby#35]
Hi, I've recently upgraded from a pre 1.0.0 version of this gem and I now receive errors when I make calls to find customers.

```
undefined method `first' for #<ChartMogul::Customer:0x007f2c416c1050>
/gems/chartmogul-ruby-1.0.0/lib/chartmogul/customer.rb:26 in find_by_external_id
```

What is the upgrade process for data which is stored in the import API?  - I simply replaced calls from the `ChartMogul::Import` namespace with the `ChartMogul` version as per the diff.

**Comment by pkopac** [comment]
Hi @stephenbinns, thank you for the report. I'll check out what the problem is shortly. The upgrade process is as you say, just change the namespace.

**Comment by pkopac** [comment]
Fixed it in version 1.0.1. Documentation is being updated, so it will be up-to-date soon. I'm sorry for the inconvenience.
